### ISSUE TITLE
Unable to edit code

### ISSUE THREAD

**Issue by zozzi2** [uzayyli/Haxball-Headless-Bot#2]
I found this code today but i cant edit it because the code is just a straight line, because of that the code is uneditable and i cannot change stuff how i want, is there any way you could make the code go down instead of go straight?
Thanks!

**Comment by zozzi2** [comment]
Thank you!

**Comment by uzayyli** [comment]
yeah i want to prevent people editing the code.. seeing some horrible exercises people do.. i would rather have it used as it is :)

**Comment by zozzi2** [comment]
Oh, no worries, I only wanted to edit the code because the language was different from mine.

**Comment by zozzi2** [comment]
Thank you :D

**Comment by uzayyli** [comment]
Just uploaded an English version, please let me know if there are any bugs :)
